playground-app/src/scenarios/index.ts

```diff
@@ -21,6 +23,8 @@ export const scenarios = {
   "Enable based on the current time": enableByTime,
   "Multi-variant experiment": pseudoRandomSplit,
   "Progressive rollout": progressRollout,
+  "Shared evaluators": sharedEvaluator,
+  "Boolean shorthand": booleanShorthand,
```

Comment:
Not necessary, but I wonder if this is better directly after the boolean flag (2nd in the list).

Reply:
I'm going to leave the order as is because it matches how we introduce these topics in the [flag definition docs](https://flagd.dev/reference/flag-definitions/).